Documentation/troubleshooting.rst

```diff
@@ -470,13 +422,26 @@ The script has the following list of prerequisites:
 * Requires ``kubectl``.
 * ``kubectl`` should be pointing to your cluster before running the tool.
 
-You can download the latest version of the cluster-diagnosis.zip file
-using the following command:
+You can download the latest version of the cilium-sysdump using the following
+command:
+
+.. code:: bash
+
+    $ curl -sLO https://github.com/cilium/cilium-sysdump/releases/download/stable/cilium-sysdump.zip
```

Comment:
`stable` should actually be replaced by the cilium-sysdump version in the URL.

Reply:
The intent is to the latest stable version assuming that cilium-sysdump is always backwards compatible. Do you see a problem with that?

Reply:
Does github actually provide a notion of a `stable` / latest release zip like this? The URL doesn't work right now.

Reply:
GitHub doesn't have a `stable` alias, but it does have a `latest`, for the release page only though. If the latest release will always be the stable one for cilium-sysdump, we can do the following:
```bash
# Retrieve ZIP URL from GitHub API:
# We need to -o cilium-sysdump because the last release currently still has the old name.
curl -sL -o cilium-sysdump.zip $(curl -sL https://api.github.com/repos/cilium/cilium-sysdump/releases/latest | jq -r '.assets[].browser_download_url')
# To avoid using jq:
curl -sL -o cilium-sysdump.zip $(curl -sL https://api.github.com/repos/cilium/cilium-sysdump/releases/latest | grep -oP '(?<="browser_download_url": ").*.zip(?=")')
```

Reply:
What's the conclusion @tgraf ?

Reply:
According to:
https://help.github.com/en/github/administering-a-repository/linking-to-releases

This will work:
https://github.com/cilium/cilium-sysdump/releases/latest/download/cilium-sysdump.zip